docs/reference/cluster/reroute.asciidoc

```diff
@@ -90,13 +92,16 @@ query parameter, which will attempt a single retry round for these shards.
 
 `routing_table`::
     Shows the `routing_table` part of the response.
-        
+
 `version`::
     Shows the cluster state version.
---
+======
+=====
+====
```

Comment:
These block delimiters are unneeded and preventing the docs build from working properly.
```suggestion
```